python/dgl/distributed/dist_context.py

```diff
@@ -44,7 +44,23 @@ def _init_rpc(ip_config, num_servers, max_queue_size, net_type, role, num_thread
 def initialize(ip_config, num_servers=1, num_workers=0,
```

Comment:
Can we use num_sampler_workers? Many users could be confused by num_worker and num_trainer.

Reply:
we could say `num_samplers`. I'm thinking if the worker processes will only be used by samplers in the future?